stdlib/public/stubs/LibcShims.cpp

```diff
@@ -50,7 +53,11 @@ SWIFT_RUNTIME_STDLIB_INTERFACE
 __swift_size_t swift::_swift_stdlib_fwrite_stdout(const void *ptr,
                                                   __swift_size_t size,
                                                   __swift_size_t nitems) {
+#if defined(__ANDROID__)
+  return __android_log_write(ANDROID_LOG_DEBUG, "SwiftRuntime", static_cast<const char *>(ptr));
```

Comment:
Just to be sure that I understand correctly, are you suggesting that I should loop on `nitems`, adding the NUL byte to each one at `size` and sending them one by one to `__android_log_write`?
In this case, I should use `strncopy` to copy each row of size `size` to a local variable named `row` sized `size + 1`, setting the NUL byte at `row[size]` and then sending this local variable to the log method.